packages/block-editor/src/components/inserter/block-patterns-tab.js

```diff
@@ -184,7 +187,7 @@ function BlockPatternsTabs( {
 	rootClientId,
 } ) {
 	const [ showPatternsExplorer, setShowPatternsExplorer ] = useState( false );
-	const categories = usePatternsCategories();
+	const categories = usePatternsCategories( rootClientId );
```

Comment:
I think there's another usage for this hook, should we pass the rootClientId there too

Reply:
We didn't really need the `rootClientId` in the other call because we were taking it into account for filtering the patterns and the calculation was for filtering the patterns based on the selected category. Having said that, I updated the other call just for consistency.